build/docs.js

```diff
@@ -16,19 +20,28 @@ async function buildDocs() {
       files.map(async (file) => {
         const doc = await readFile(file, "utf-8");
         const content = fm(doc);
+        const splitName = file.split("/");
+        let fname = splitName[splitName.length - 1]
+        fname = fname.replace(".md","")
         return {
           ...content,
           html: mk(content.body, { breaks: true }),
-          path: path.relative(path.resolve(__dirname, "../src/docs/"), file),
+          path: path.relative(path.resolve(__dirname, mdDir), file),
+          name:fname
```

Comment:
If you use the front-matter tags in the markdown file you can specify a name there without having to parse out the filename. It will be part of the generated content.